### PR TITLE
Implemented the 'XmlWriter' class

### PR DESCRIPTION
**Pull request by nicoschurig**
- #6 according to #1
- the functionality was manually tested (see: 87c6a88fbdd50e0351b3a9468dea6f8f5cadeb4a)

### CODE REVIEW

File: OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs [hunk 1]
```diff
@@ -1,15 +1,107 @@
...
 using System.Xml;
+using System.Globalization;
 
 namespace OpenMensa_Parser
 {
-    public class XmlWriter
+    static class XmlWriter
     {
-        public string XmlFilePath {get; private set;}
+        public static string fileName = "OpenmenMensaXml.xml";
```
Comment (tuning-8) [inline OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs:10]: - please change the fixed filename to be passed by a constructor

File: OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs [hunk 2]
```diff
@@ -1,15 +1,107 @@
 using System;
+using System.Text;
 using System.Xml;
+using System.Globalization;
 
 namespace OpenMensa_Parser
 {
-    public class XmlWriter
+    static class XmlWriter
```
Comment (tuning-8) [inline OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs:8]: - please add a constructor and change the class to non-static

File: OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs [hunk 3]
```diff
@@ -1,15 +1,107 @@
...
     {
-        public string XmlFilePath {get; private set;}
+        public static string fileName = "OpenmenMensaXml.xml";
+        public static string parserVersion = "1.01-1";
+        public static string feedInformation = "http://openmensa.org/open-mensa-v2";
+        public static string schemaInstance = "http://www.w3.org/2001/XMLSchema-instance";
+        public static string schemaLocation = "http://openmensa.org/open-mensa-v2.xsd";
```
Comment (tuning-8) [inline OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs:14]: - please add the strings above to the constructor to be more flexible

File: OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs [hunk 4]
```diff
@@ -1,15 +1,107 @@
...
+            WriteOpenMensaStandardInformation(xmlWriter);
+            WriteMenuInformation(xmlWriter);
+        }
+
+        public static void WriteOpenMensaStandardInformation(XmlTextWriter xmlWriter)
+        {            
+            //Writing process:
+            xmlWriter.WriteStartDocument();                 //writes: <?xml version="1.0" encoding="utf-8"?>
+            xmlWriter.WriteStartElement("openmensa");
+            xmlWriter.WriteAttributeString("version", "2.1");
```
Comment (tuning-8) [inline OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs:37]: - should also be a variable version (e.g. might change to 2.2 by the OpenMensa organization)

**Review by tuning-8 — CHANGES_REQUESTED** [review]
- please consider implementing a constructor for the XmlWriter class to be more flexible for changes
- I have marked all the lines that I think should be included in the constructor

**Comment by nicoschurig** [comment]
- @tuning-8  thank you for your feedback!
- I implemented all changes requested.

**Todo:**
- Adapt the #1  to match the new _XmlWriter_ class.

**Comment by nicoschurig** [comment]
- I forgot to implement the constructor, now everything should be fine.

**Pull request merged**